server/src/core/api-rest/api-rest-query-builder/factories/parsers/limit-parser.factory.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+
+import { Request } from 'express';
+
+@Injectable()
+export class LimitParserFactory {
+  create(request: Request) {
+    const limitQuery = request.query.limit;
+    if (typeof limitQuery !== 'string') {
+      return 60;
+    }
+    const limitParsed = parseInt(limitQuery);
+    if (!Number.isInteger(limitParsed)) {
```

Comment:
Should be number after the other suggestion
```suggestion
    if (limitParsed) {
```